modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -647,4 +721,5 @@ sda.enkf.multisite <- function(settings,
     
   } ### end loop over time
   
-} # sda.enkf
\ No newline at end of file
+ } # sda.enkf
+}
```

Comment:
Unclear where the extra } is coming from. The comment indicates the close of the function, so presumably is now in the wrong place, and there needs to be an additional comment indicating what the new } is closing

Reply:
looking for a quick fix & response here

Reply:
So the `}` at line 725 corresponds to the `{` at line 42; `}` at line 721 corresponds to the `{` at line 322; and the `}` at line 719 corresponds to the `{` at line 342.
The key to understanding the structures is that there is an unclosed `else` control at line 322 if you pay attention to it.

Reply:
Which is not written by me but throws errors when executing it.

Reply:
Fixed.